selfservice/strategy/oidc/provider.go

```diff
@@ -65,3 +65,14 @@ func (c *Claims) Validate() error {
 	}
 	return nil
 }
+
+func UpstreamParameters(provider Provider, sentProviders map[string]string) ([]oauth2.AuthCodeOption, error) {
```

Comment:
`sentProviders` doesn't seem to be a good name. I wouldn't understand what to do with this function.

Reply:
good point, I think this might've been autofilled by co-pilot :)